source/develop/troubleshooting-nfs-storage-issues.html.md

```diff
@@ -166,3 +166,13 @@ In this case it is a good idea to set the tcp window parameters on the NFS serve
 To activate these settings for the running server reload them with
 
        # sysctl -p 
+
+### Permissions issues
+
+On RHEL 7, starting the `nfs-server` service changes back the ownership of the directory from `vdsm:kvm` to `root:root`. 
+Rerun the `chown` command **after** starting the service:
```

Comment:
Never seen this, but not sure why do you need to use ```/storage```. Typically you have something
like ```/export/``` owned by root, and you create a directory like ```/export/domain1/``` for every domain,
and chown ```/export/domain1/```, not ```/export/```.

Regardless this does not look like a good advise, as the permissions will change each time
you restart the server (or maybe when you start it after boot?). This is not a usable setup.